numpy/f2py/__init__.py

```diff
@@ -2,7 +2,7 @@
 """Fortran to Python Interface Generator.
 
 """
-__all__ = ['run_main', 'compile', 'get_include', 'f2py_testing']
+__all__ = ['run_main', 'compile', 'get_include']
```

Comment:
The `f2py_testing` module is still present in the namespace, so this shouldn't break any backwards compatibility.